README.md

```diff
@@ -82,6 +82,12 @@ docker run -it --rm ,
     appwrite/appwrite:0.10.4
 ```
 
+### Stop The Running Docker Containers (Windows/Unix)
+
+```bash
+docker-compose -f docker-compose.yml down
+```
```

Comment:
I don't think everyone using appwrite will have installed it on their system. So, to make this work they will have to download the yml file and then run docker-compose. Is there some workaround for this?

Reply:
Hey but I simply run the docker command and it automatically started the docker-compose. So I thought why not add a script to put it down. Cause a lot many containers were running simultaneously, and it would be a lot of pain to stop them individually.

Reply:
Right I understand that. Docker-compose is run during the installation. But the user might not have that yml file in that directory. Maybe we can point some additional comments about the location of the yml file to give a better understanding. I have proposed a similar solution here and stopping like this works for me. https://github.com/appwrite/appwrite/issues/1837

Reply:
Okay thank you, I will look into this.

Reply:
I checked your command however in my terminal it says `zsh: no matches found: name=appwrite*` . So I modified my previous command and now it works fine. Please do check it.

Reply:
I think this is because it could not find any running containers. Did you confirm the appwrite containers running when you ran this command?
However, I your command also works on bash.
`docker container stop $(docker ps | grep "appwrite*" | cut -d " " -f 1)`

Reply:
Yes at the moment when I ran your script, the containers were running.